js/app.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+let image1 = document.querySelector('section img:first-child');
+let image2 = document.querySelector('section img:nth-child(2)');
+let image3 = document.querySelector('section img:last-child');
+
+let resultsButton = document.getElementById('results');
+let index1 = 0;
+let index2 = 0;
+let index3 = 0;
+let clicks = 0;
+
+
+function Odd(name, src) {
+  this.name = name;
+  this.src = src;
+  this.clicks = 0;
+  this.views = 0;
+}
+
+let odd1 = new Odd('bag', './images/bag.jpeg');
+let odd2 = new Odd('banana', './images/banana.jpeg');
+let odd3 = new Odd('bathroom', './images/bathroom.jpeg');
+let odd4 = new Odd('boots', './images/boots.jpeg');
+let odd5 = new Odd('breakfast', './images/breakfast.jpeg');
+let odd6 = new Odd('bubblegum', './images/bubblegum.jpeg');
+let odd7 = new Odd('chair', './images/chair.jpeg');
+let odd8 = new Odd('cthulhu', './images/cthulhu.jpeg');
+let odd9 = new Odd('dog-duck', './images/dog-duck.jpeg');
+let odd10 = new Odd('dragon', './images/dragon.jpeg');
+let odd11 = new Odd('pen', './images/pen.jpeg');
+let odd12 = new Odd('pet-sweep', './images/pet-sweep.jpeg');
+let odd13 = new Odd('scissors', './images/scissors.jpeg');
+let odd14 = new Odd('shark', './images/shark.jpeg');
+let odd15 = new Odd('sweep', './images/sweep.png');
+let odd16 = new Odd('tauntaun', './images/tauntaun.jpeg');
+let odd17 = new Odd('unicorn', './images/unicorn.jpeg');
+let odd18 = new Odd('water-can', './images/water-can.jpeg');
+let odd19 = new Odd('wine-glass', './images/wine-glass.jpeg');
+
+
+let odds = [odd1, odd2, odd3, odd4, odd5, odd6, odd7, odd8, odd9, odd10, odd11, odd12, odd13, odd14, odd15, odd16, odd17, odd18, odd19];
+
+let firstOdd;
+let secondOdd;
+let thirdOdd;
+
+function getRandomIndex() {
+  return Math.floor(Math.random() * odds.length);
+}
+
+
+function renderOdds() {
+  index1 = getRandomIndex();
+  index2 = getRandomIndex();
+  index3 = getRandomIndex();
+
+  firstOdd = odds[index1];
+  secondOdd = odds[index2];
+  thirdOdd = odds[index3];
+
+  image1.src = firstOdd.src;
+  image1.alt = firstOdd.name;
+  image1.title = firstOdd.name;
+  image1.id = index1;
```

Comment:
None of your id's are properly reassigning on the deployed site. Looks like no matter which image I click only three objects can receive clicks.